addons/web/static/src/js/widgets/rainbow_man.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.rainbow_man', function (require) {
+"use strict";
+
+var ajax = require("web.ajax");
+var Widget = require('web.Widget');
+var core = require('web.core');
+var qweb = core.qweb;
+var _t = core._t;
+
+ajax.loadXML("/web/static/src/xml/rainbow_man.xml", qweb);
```

Comment:
remove this, static xml is automatically loaded in web client (look at __manifest__.py)